### ISSUE TITLE
No docker.exe file after upgrading Golem from 0.19.1 to 0.19.2 on Windows

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#4204]
## Description

**Golem Version**: 0.19.2 build  https://github.com/golemfactory/golem/releases/tag/untagged-ab3fcdf1b58dc9340392

**Golem-Messages version** (leave empty if unsure):

**Electron version** (if used): 0.19.1

**OS** [e.g. Windows 10 Pro]: Windows 10 Pro

**Branch** (if launched from source): - 

**Mainnet/Testnet**: Mainnet

**Description of the issue**:

After upgrading Golem from 0.19.1 to 0.19.2 Golem is not starting. There is no `docker.exe` file in Program Files/Golem. Docker machine won't be created and Golem is not starting. It happens with upgrade only. Clean installation ends in success, Golem connect to nodes, `docker.exe` is present.

**Actual result**:

Docker is not available, it is not possible to compute on Golem.

**Screenshots**:

_If applicable, add screenshots to help explain your problem._

## Steps To Reproduce
_Short description of steps to reproduce the behavior:_
e.g.
1. Machine with preinstalled Golem 0.19.1 is needed.
2. Download installer from  https://github.com/golemfactory/golem/releases/tag/untagged-ab3fcdf1b58dc9340392
3. Run installator, go trough whole installation process, restart machine
4. Launch Golem
5. Check in logs for error

## Expected behavior

Golem should launch without problems and connect to nodes after upgrade.

## Logs and any additional context

```
2019-05-16 18:17:06 ERROR    golem.docker.manager                
                ***************************************************************
                Docker is not available, not building images.
                Golem will not be able to compute anything.
                Command 'docker info' returned CalledProcessError(127, '[WinError 2] The system cannot find the file specified')
                ***************************************************************
                
2019-05-16 18:17:06 ERROR    golem.node                          
                    There was a problem setting up the environment: Docker unavailable.
                    Golem will run with limited functionality to support
                    communication with local clients.
                    
2019-05-16 18:17:09 INFO     golem.node                          Waiting for password to unlock the account. Run `golemcli account unlock` and enter your password.

```

```
C:\Users\ederenn>docker
'docker' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\ederenn>
```

installation logs for an update: 
[MSI7968.LOG](https://github.com/golemfactory/golem/files/3191202/MSI7968.LOG)

## Proposed Solution?
_(Optional: What could be a solution for that issue)_

**Comment by etam** [comment]
What we've found so far:
- `docker.exe` and `docker-machine.exe` are both present in the installation package
- installer removed `docker.exe` and did not extract new one.

Interesting part from the log file:
```
MSI (c) (74:90) [03:59:37:172]: Component: dockermachine.exe; Installed: Absent;   Request: Local;   Action: Local
MSI (c) (74:90) [03:59:37:172]: Component: docker.exe; Installed: Absent;   Request: Local;   Action: Null
```

`Action: Null` appears only next to `docker.exe`

**Comment by maaktweluit** [comment]
Resolved by #4205
